client/components/credit-card/README.md

```diff
@@ -43,22 +42,22 @@ Example usage within `<CreditCard>`:
 import StoredCard from 'components/stored-card';
 
 export default ( cardData, onRemoveCard ) => {
-    return (
-        <CreditCard>
-            <StoredCard { ...cardData } />
-            <Button onClick={ onRemoveCard }>Remove</Button>
-        </CreditCard>
-    );
+	return (
+		<CreditCard>
+			<StoredCard { ...cardData } />
+			<Button onClick={ onRemoveCard }>Remove</Button>
+		</CreditCard>
+	);
 };
 ```
 
 ### Props
 
 All props are required.
 
-Name | Type | Description
--- | -- | --
-`lastDigits` | `string` | Last digits of card number.
-`cardType` | `string` | Card type (e.g. `visa`).
-`name` | `string` | Name on card (e.g. `James Smith`).
-`expiry` | `string` | Expiration date in MM/YY form.
+| Name         | Type     | Description                                               |
+| ------------ | -------- | --------------------------------------------------------- |
+| `lastDigits` | `string` | Last digits of card number.                               |
+| `cardType`   | `string` | Card type (e.g. `visa`).                                  |
+| `name`       | `string` | Name on card (e.g. `James Smith`).                        |
+| `expiry`     | `string` | Expiration date in ISO 8601 (YYYY-MM or YYYY-MM-DD) form. |
```

Comment:
This line was inaccurate and updated. The rest of the changes in this file are prettier autoformatting.